beoordelingstool_dockwidget_base.ui

```diff
@@ -97,7 +97,7 @@
         </rect>
        </property>
        <property name="text">
-        <string>Upload json</string>
+        <string>Upload zip and json</string>
```

Comment:
Hier wordt toch alleen de zip geüploaded? En niet ook de json? (Hoewel de json misschien in de zip zit, maar goed)